tools/depends/target/webos-wayland-extensions/Makefile

```diff
@@ -0,0 +1,41 @@
+-include ../../Makefile.include
+include WEBOS-WAYLAND-EXTENSIONS-VERSION
+DEPS = WEBOS-WAYLAND-EXTENSIONS-VERSION Makefile ../../download-files.include
+
+ifeq ($(CROSS_COMPILING), yes)
+  DEPS += ../../Makefile.include
+else
+  ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+  ifeq ($(PLATFORM),)
+    PLATFORM = native
+    TARBALLS_LOCATION = $(ROOT_DIR)
+  endif
+endif
+
+RETRIEVE_TOOL := curl -Ls --create-dirs --retry 10 --retry-delay 3
```

Comment:
This shouldn't be needed (only needed for native builds) as it's set in `../../Makefile.include`